zuul.d/projects.yaml

```diff
@@ -498,3 +498,12 @@
     name: github.com/pycontribs/selinux
     templates:
       - system-required
+
+- project:
+    name: github.com/project-receptor/receptor
+    default-branch: devel
+    templates:
+      - system-required
+      - ansible-python-jobs
+      - ansible-python3-jobs
+      - publish-to-pypi
```

Comment:
If you are not gating, these project-templates are not going to work. As they have gate jobs, so for this, we need to have a discussion

Reply:
Yeah you are right. I missed that. Let me update this for now and add them as the following for now.

```
 third-party-check:
      jobs:
        - ansible-tox-linters
        - ansible-tox-py36
        - ansible-tox-py37
```

Reply:
done